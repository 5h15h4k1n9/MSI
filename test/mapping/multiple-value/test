store(1, "Alice")
store(2, "Charlie")
read(1)
read(2)
exit
