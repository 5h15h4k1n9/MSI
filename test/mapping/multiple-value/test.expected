(void) void
(void) void
(string) "Alice"
(string) "Charlie"
Bye-bye!
